.vscode/settings.json

```diff
@@ -14,7 +14,7 @@
     "CLANG_DIR": "${workspaceFolder}/polygeist/llvm-project/build/lib/cmake/clang",
     "CMAKE_C_COMPILER": "clang",
     "CMAKE_CXX_COMPILER": "clang++",
-    "LLVM_ENABLE_LLD": "ON",
+    "LLVM_USE_LINKER": "lld",
```

Comment:
I updated this option just in case, because `LLVM_ENABLE_LLD` and `LLVM_USE_LINKER` cannot co-exist